scripts/badger/registry.py

```diff
@@ -2,20 +2,99 @@
 from great_ape_safe import GreatApeSafe
 from helpers.addresses import get_registry as registry
 
-console = Console()
+C = Console()
 
 # assuming techOps
-contract_registry = registry()
-chain_safe = GreatApeSafe(contract_registry.badger_wallets.techops_multisig)
-chain_safe.init_badger()
+r = registry()
+SAFE = GreatApeSafe(r.badger_wallets.techops_multisig)
+SAFE.init_badger()
+
+# Allowed metadata behaviors
+# NOTE: Should be kept up to date matching the behaviors defined on the SDK at
+# https://github.com/Badger-Finance/badger-sdk/blob/main/src/vaults/enums/vault-behavior.enum.ts
+ALLOWED_BEHAVIORS = [
+  'Compounder',
+  'Helper',
+  'DCA',
+  'Ecosystem',
+  'Ecosystem Helper',
+  'None'
+]
+
+# Add any vaults's addresses that you want to promote or demote
+VAULTS_ARRAY = []
+# Add the new statuses you want to promote or promote each vault to
+NEW_STATUS_ARRAY = []
+
+# promote a batch of vaults given their addresses and new status
+def promote_vaults():
+    if VAULTS_ARRAY and NEW_STATUS_ARRAY:
+        for vault in VAULTS_ARRAY:
+            info = SAFE.badger.registry_v2.productionVaultInfoByVault(vault)
+            new_status = NEW_STATUS_ARRAY[VAULTS_ARRAY.index(vault)]
+            if info[2] < new_status:
+                print(info)
+                SAFE.badger.promote_vault(info[0], info[1], info[3], new_status)
+                assert SAFE.badger.registry_v2.productionVaultInfoByVault(vault)[2] == new_status
+                C.print(f"[green]Promoting {vault} to {new_status}[/green]")
+            else:
+                C.print(f"[red]Promoting {vault} to lower state, current state is {info[2]}[/red]")
```

Comment:
this is a bit confusing since it is not _actually_ promoting.

makes we wonder if an `update_status` func or something would actually be more efficient than the two current `promote_vaults` and `demote_vaults` funcs.